src/System.Management.Automation/help/CommandHelpProvider.cs

```diff
@@ -540,6 +551,41 @@ private string GetHelpFile(string helpFile, CmdletInfo cmdletInfo)
             return location;
         }
 
+        private string CalculateHelpSearchPath(CmdletInfo cmdletInfo)
```

Comment:
I don't see this method used anywhere.

Reply:
Good catch, got refactored into HelpUtils.cs